docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+Visual Studio Code has 100s of [settings](/docs/getstarted/settings.md), 1000s of [extensions](/docs/editor/extension-marketplace.md), and unnumerable ways to adjust the UI layout to customize the editor. VS Code **Profiles** let you create sets of customizations and quickly switch between them or share them with others. This topic explains how to create, modify, export, and import profiles.
+
+## Create a Profile
+
+VS Code treats your current configuration as the **Default Profile**. As you modify settings, install extensions, or open and close views and panels, these customizations are tracked in the Default Profile.
```

Comment:
I would not say open and close views and panels instead say change views layout like moving views. Because opening and closing views is not part of the profile.